pkl-commons-cli/src/main/kotlin/org/pkl/commons/cli/CliTestOptions.kt

```diff
@@ -17,4 +17,9 @@ package org.pkl.commons.cli
 
 import java.nio.file.Path
 
-class CliTestOptions(val junitDir: Path? = null, val overwrite: Boolean = false)
+class CliTestOptions(
+  val junitDir: Path? = null,
+  val overwrite: Boolean = false,
+  val junitAggregateReports: Boolean = false,
+  val junitSuiteName: String? = null,
+)
```

Comment:
How about setting the default name as `pkl-tests`?

Also, let's make `junitSuiteName` non-nullable, and declare the default here.

Also, maybe call it `junitAggregatedTestSuitesName"? This is the name of the `<testsuites>`,  not the `<testsuite>`.

```kt
class CliTestOptions(
  val junitDir: Path? = null,
  val overwrite: Boolean = false,
  val junitAggregateReports: Boolean = false,
  val junitAggregatedTestSuitesName: String = "pkl-tests",
) {
  companion object {
    val defaults = CliTestOptions()
  }
}
```

We should also make the flags in Gradle and the CLI non-optional (use `.convention` in Gradle, `.default` in Clikt), and refer to the defaults here.

Reply:
Done.